src/elements/content-sidebar/__tests__/ActivitySidebar.test.js

```diff
@@ -1496,35 +1501,220 @@ describe('elements/content-sidebar/ActivitySidebar', () => {
+                        target: {
+                            avatar_url: '',
+                            id: '2',
+                            name: 'Beyonce',
+                            type: 'user',
+                        },
+                        type: 'task_collaborator',
+                    },
+                ],
+                limit: 10,
+                next_marker: null,
+            },
+            created_at: '2019-01-01',
+            created_by: {
+                target: {
+                    id: '000',
+                    role: 'CREATOR',
+                    status: 'NOT_STARTED',
+                    target: {
+                        id: '100',
+                        name: 'Jay-Z',
+                    },
+                    type: 'task_collaborator',
+                },
+            },
+            due_at: '2019-02-02',
+            id: '0',
+            permissions: {
+                can_update: false,
+                can_delete: false,
+                can_create_task_collaborator: false,
+                can_create_task_link: false,
+            },
+            status: 'NOT_STARTED',
+            task_links: {
+                entries: [
+                    {
+                        id: '03',
+                        permissions: {
+                            can_delete: false,
+                            can_update: false,
+                        },
+                        target: {
+                            id: '4',
+                            type: 'file',
+                        },
+                        type: 'task_link',
+                    },
+                ],
+                limit: 1,
+                next_marker: null,
+            },
+        };
+
+        const expectedVersionItem = {
+            id: '1060370614597',
+            authenticated_download_url: 'https://someurl.com',
+            created_at: '2022-07-06T03:01:28-07:00',
+            extension: 'boxnote',
+            is_download_available: true,
+            modified_at: '2022-07-06T03:01:28-07:00',
+            modified_by: {
+                id: '18836063940',
+                login: 'jdoe@box.com',
+                name: 'John Doe',
+                type: 'user',
+            },
+            name: 'Title',
+            permissions: {
+                can_annotate: false,
+                can_create_annotations: false,
+                can_delete: true,
+                can_download: true,
+                can_preview: true,
+                can_view_annotations: false,
+                can_view_annotations_all: true,
+                can_view_annotations_self: true,
+            },
+            restored_at: null,
+            restored_by: null,
+            retention: null,
+            size: 4159,
+            trashed_at: null,
+            trashed_by: null,
+            type: 'file_version',
+            uploader_display_name: 'John Doe',
+            version_number: undefined,
+        };
```

Comment:
These expected values look like they may be copied from the fixture. I wonder if instead of copying, if the expected variables here could be set as deep clones from the fixture? I think this would help with duplication while avoiding the concerns with potential side effects and mutability of the objects.

Reply:
Hm yeah, let me see if I can deep clone these specific fields from the fixture; if not, I can try copying it over from the fixture to help with duplication!

Reply:
Hmm, have yet to find a way too deep clone specific properties from an object; unfortunately some of the items aren't exactly the same as the ones in the fixture - if mutability is a concern, do you think something like this would suffice?
```
const expectedAnnotationOpen = {
   created_at: annotationOpen.source.annotation.created_at,
   created_by: annotationOpen.source.annotation.created_by,
   id: annotationOpen.source.annotation.id,
   modified_at: annotationOpen.source.annotation.modified_at,
   permissions: annotationOpen.source.annotation.permissions,
   status: annotationOpen.source.annotation.status,
 };

Reply:
As discussed, using something like `expectedVersionItem = cloneDeep(filterableActivityFeedItems.versionItem.source)` and cloning specific properties worked for the tests!